source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -128,9 +129,30 @@ const SummaryList: React.FC<Props> = ({
   help,
   editable,
 }) => {
-  // For development: show an error popup when something is configured wrong.
-  const showNotification = useNotification();
+  const [isModified, setModified] = useState(false);
 
+  const sortAnswers = (listItems: SummaryListItem[]) => {
+    listItems.forEach((item) => {
+      if (ArrayType.includes(item.type)) {
+        const category = categories?.find(
+          (categoryItem) => item.category === categoryItem.category
+        );
+        const sortField = category?.sortField;
+        if (sortField && Array.isArray(answers[item.id])) {
+          const answerCopy = [...answers[item.id]].sort(
+            (valueA: Answer, valueB: Answer) => {
+              const a = Number(valueA[sortField]);
+              const b = Number(valueB[sortField]);
+              return a - b;
+            }
+          );
+          answers[item.id].splice(0);
```

Comment:
```typescript
answers[item.id].splice(0);
answers[item.id].push(...answerCopy);
```

I'm not sure what this is supposed to do. `splice(0)` essentially empties the array, and then we push a single element to it? Wouldn't `answers[item.id] = answerCopy` do the same thing?

However on the larger scale, it's very cryptic that `sortAnswers` sorts the array in place. I would prefer if we stick to a functional approach here: `const newListItems = sortAnswers(listItems)`.

Reply:
Yes I know but at the same time re-assignment is a bad practise so then I have to:
          // eslint-disable-next-line no-param-reassign
          answers[item.id] = answerCopy;

There is a reason why it has to be sorted in place, the architecture is built upon direct editing of the answers. Lets talk about it